### PR TITLE
Remove "stable" scrollbar-gutter

### PR DESCRIPTION
**Pull request by exacs**
This PR removes the property `scrollbar-gutter` that causes visual bugs in Chrome-related browsers.

- `scrollbar-gutter` is a property that keeps a "white rail" on the right side of the window when there is no scrollbar.
- KPM uses `left: 0; right: 0` to calculate its width.
- Chrome apparently has a bug where `right: 0` is calculated from the right corner of the window, _including_ the gutter

See this screenshot for details:

Captura de pantalla 2024-06-25 a las 13 48 01

[AB#3889](https://dev.azure.com/kth-integration/f3bf8cac-2bdc-4cc2-92e4-2ab44dcc461c/_workitems/edit/3889)

### CODE REVIEW

**Comment by ahrsjo** [comment]
AB#3889

**Comment by ahrsjo** [comment]
AB#3889

**Review by ahrsjo — APPROVED** [review]
giphy

**Review by EllenWiderstrand — CHANGES_REQUESTED** [review]
As discussed on Slack: The "Logga in" button is jumping (it is first displayed to the left and then to the right where it is supposed to be)

https://github.com/user-attachments/assets/6e79f947-d860-4846-a00c-30eb17c3d580

File: @kth/style/scss/components/kpm.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@
 @use "../tokens/colors.scss";
 @use "../tokens/sizing.scss";
 
+// This file contains the minimal CSS needed to avoid layout shifts and other
```
Comment (EllenWiderstrand) [inline @kth/style/scss/components/kpm.scss:6]: Nice!

**Review by EllenWiderstrand — APPROVED** [review]

**Pull request merged**